_sass/bootstrap/_button-groups.scss

```diff
@@ -27,7 +27,7 @@
   .btn + .btn-group,
   .btn-group + .btn,
   .btn-group + .btn-group {
-    margin-left: -1px;
```

Comment:
Please do not alter vendor files (this makes upgrading external dependencies harder). If you have to override such stuff, create a custom CSS class in our files and use this class instead.